src/main/msp/msp.c

```diff
@@ -3002,6 +3003,17 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
         }
         break;
 
+    case MSP2_SEND_DSHOT_COMMAND:
+        {
+#ifdef USE_DSHOT
+            const uint8_t motorIndex = sbufReadU8(src);
+            const uint8_t commandIndex = sbufReadU8(src);
+            const uint8_t commandType = sbufReadU8(src);
+            dshotCommandWrite(motorIndex, getMotorCount(), commandIndex, commandType);
```

Comment:
For some ESCs (KISS at least), configuration commands and save commands must be sent without any other Dshot signals in between. So it probably makes sense to extend this a bit and allow an array (length sent first) of commands to be sent, and in the case of blocking type commands disable the Dshot output before starting to send commands.
Thoughts?

Reply:
@mikeller makes sense. I did not understand how do i "disable Dshot output"?

Reply:
Simple (see `cliDshotProg()` in `cli/cli.c` for an example):

```
motorDisable();
<send commands>
motorEnable();
```

Also, come to think of it, we probably want to check if we are disarmed before we send Dshot commands. :wink: